cluster/gce/configure-vm.sh

```diff
@@ -949,7 +949,7 @@ EOF
   if [[ ! -z "${KUBELET_APISERVER:-}" ]] && [[ ! -z "${KUBELET_CERT:-}" ]] && [[ ! -z "${KUBELET_KEY:-}" ]]; then
     cat <<EOF >>/etc/salt/minion.d/grains.conf
   kubelet_api_servers: '${KUBELET_APISERVER}'
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
What you described is the desired behavior. If we were using the /24 we wouldn't have the bug this is trying to fix (or, we would be hitting it on the first container on any node). 

It's possible we started hitting some race condition when we moved the master to gci where the cni config got written out with the wrong pod-cidr and we just didn't care till now. 

the kubelet sets pod cidr straight out the gate: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L645, and we don't really allow it to change because that requires a restart of all containers on the bridge. 

I didn't dig too deep into the regression but vaguely remember the --pod-cidr reflecting in the bridge even in 1.3